libs/angular-routing/src/lib/link-active.directive.ts

```diff
@@ -15,6 +15,7 @@ import { LinkTo } from './link-to.directive';
 import { Router } from './router.service';
 import { combineLatest, of, Subject, Subscription } from 'rxjs';
 import { map, mapTo, startWith, takeUntil } from 'rxjs/operators';
+import { filterNullable } from './operators';
```

Comment:
Let's not use barrels internally 

```suggestion
import { filterNullable } from './operators/filter-nullable.operator';
```